archimedes/src/app/component/image/image.component.html

```diff
@@ -0,0 +1,67 @@
+<div *ngFor="let image of images" class="col-sm-6 col-md-4 col-xl-3">
+    <div class="card mb-3">
+        <a href="{{image.download_url}}" target="_blank">
+            <img class="card-img-top" src="{{image.thumbnail_url}}" alt="{{image.location}}">
+        </a>
+        <div class="card-block">
+
+            <p class="card-text" (click)="(selectedImage === image.id) ? selectedImage = 0 : selectedImage = image.id">
+                View Image Details
+            </p>
+
+            <ng-container *ngIf="selectedImage === image.id">
+
+                <p class="card-text" *ngIf="image.tags && image.tags.length">
+                    <strong>Tags: </strong>
+                    <ng-container *ngFor="let tag of image.tags">
+                        <a [href]="['https://ucf.tandemvault.com/browse?tags=' + tag]" target="_blank">{{tag}}</a>,
```

Comment:
Can this loop be updated to not print the trailing comma for the last item in the list?